bin/install-ci-tools

```diff
@@ -6,7 +6,7 @@ if [[ -z "${KIND_VERSION}" ]]; then
 fi
 
 if [[ -z "${HELM_VERSION}" ]]; then
-  export HELM_VERSION="3.2.1"
+  export HELM_VERSION="3.5.4"
```

Comment:
FWIW helm 3.6.0 just came out, and [it has a useful feature that we may be able to use in our purest setup](https://github.com/rikatz/kubepug/issues/139#issuecomment-850050037), so I'd recommend making an attempt to adopt it.

Reply:
Good call, giving it a shot -- just pushed the comment